### DISCUSSION TITLE
SMTP settings to send email error

### DISCUSSION THREAD

**Discussion by henjoe** [frangoteam/FUXA#1237] [Q&A]
Hi, I wonder who already use the SMTP settings of Fuxa? 
I tried to use it but I am having failure, it says: 
"https failure response for http://localhost:1881/api/sendmail:400 bad request"

Is my setting is wrong? (see below)
![image](https://github.com/frangoteam/FUXA/assets/49782655/897ff81a-6a12-4708-80c3-3b04fd10eb87)

Hope someone can help me. Thanks!

**Comment by henjoe** [comment]
Changing the port to 587 solves the problem.

**Comment by taylor172002** [comment]
![Capture d'écran 2025-06-09 111634](https://github.com/user-attachments/assets/8b052473-5b8a-4190-8b96-1d33e9043e8f)
Hello, I still can't connect using Sendgrid, even though the settings seem correct and work in other apps.
I'm using the NPM installation.
